src/controllers/index.js

```diff
@@ -32,6 +32,7 @@ Controllers.mods = require('./mods');
 Controllers.sitemap = require('./sitemap');
 Controllers['404'] = require('./404');
 Controllers.errors = require('./errors');
+Controllers.handleOSDRequest = require('./osd');
```

Comment:
To conform, I'd move this up two lines (for 404 and error are at the bottom), and change the property name to `osd`